vllm/config.py

```diff
@@ -1340,6 +1362,12 @@ def __post_init__(self) -> None:
         self.world_size = self.pipeline_parallel_size * \
             self.tensor_parallel_size
 
+        self.data_parallel_size = envs.VLLM_DP_SIZE
+        self.data_parallel_rank = envs.VLLM_DP_RANK
+        self.data_parallel_master_ip = envs.VLLM_DP_MASTER_IP
+        self.data_parallel_master_port = envs.VLLM_DP_MASTER_PORT
```

Comment:
Note that I'm hitting issues like:
```
RuntimeError: The server socket has failed to listen on any local network address. port: 29500, useIpv6: 0, code: -98, name: EADDRINUSE, message: address already in use
```

This is true even if I change the master port with `torchrun --master-port ...`. Currently hacking around it by changing this to `self.data_parallel_master_port = envs.VLLM_DP_MASTER_PORT + 1`

Reply:
that's strange. I also met it once but then it disappeared.

Reply:
it seems this disappeared when i remove torchrun in [af53b4b](https://github.com/vllm-project/vllm/pull/13591/commits/af53b4b40de9355c58da2e9eb588becbefc2e062)